pkg/sourcereader/local.go

```diff
@@ -30,7 +31,13 @@ func (r LocalSourceReader) GetModuleInfo(modPath string, kind string) (modulerea
 	}
 
 	reader := modulereader.Factory(kind)
-	return reader.GetInfo(modPath)
+	mi, err := reader.GetInfo(modPath)
+	if idx := strings.Index(modPath, "/community/modules/"); idx != -1 {
+		mi.RequiredApis = defaultAPIList(modPath[idx+1:])
+	} else if idx := strings.Index(modPath, "/modules/"); idx != -1 {
+		mi.RequiredApis = defaultAPIList(modPath[idx+1:])
```

Comment:
Nit: since defaultAPIList is now used by embedded and local, should it be moved to sourcereader.go instead of embedded.go? (I'm ok with not moving it if you think it is better this way).

Reply:
I had same thought:

- rename defaultRequiredAPIs
- move to sourcereader.go

but wanted a simple commit for purpose of later / clearer reversion when there is more automation here.

I will merge this while noting that we should consider the above if/when we touch this more (e.g. a `GitSourceReader` implementation).